gaia-7005/genesis.json

```diff
@@ -0,0 +1,8279 @@
+{
+  "genesis_time": "2018-08-06T08:00:00.123456789Z",
+  "chain_id": "gaia-7005",
+  "consensus_params": {
+    "block_size_params": {
+      "max_bytes": "22020096",
+      "max_txs": "10000",
+      "max_gas": "-1"
+    },
+    "tx_size_params": {
+      "max_bytes": "10240",
+      "max_gas": "-1"
+    },
+    "block_gossip_params": {
+      "block_part_size_bytes": "65536"
+    },
+    "evidence_params": {
+      "max_age": "100000"
+    }
+  },
+  "validators": [
+    {
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "tU8IqlII8g8ngcmtMVG/k/8SmunfJJBkAoOisN+qcKE="
+      },
+      "power": "119",
+      "name": "JB-The-Valid"
+    },
+    {
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "RBuRRNEzA9RA1Wrdi9PPFQJ29/n/bqN9O2tQv9Gq248="
+      },
+      "power": "2161",
+      "name": "skoed-validator-7000"
+    },
+    {
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "F2wCn9rKafNZsYZwoLGkSQIpr3rk86cjYyuhSjsjRaE="
+      },
+      "power": "255",
+      "name": "ironfork"
+    },
+    {
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "NQX4yKpOztKrmgBhGIC5WOALOLOq3LTpbzsN4ZLXGec="
+      },
+      "power": "1100",
```

Comment:
Why do we have a power already associated to a validator? Is this how much stake they had at the time the testnet crashed? We don't start fresh for a new testnet? or we have to have a certain amount of stake distributed to get the tesetnet online?

Reply:
The power fields should be the amount of steak they had when the testnet crashed. We don't start anew since people want to have power from one testnet transfer to the next. If someone has low steak but was online the entire time, they are typically given more steak.